communication/templates/lobby.html

```diff
@@ -849,6 +860,24 @@
                         }
                     }
                 }
+            } else if (ready.gameState === "ongoing") {
+                // The wordHints are empty if the round is not started (no word choosen)
+                if (ready.wordHints === null) {
+                    for (let i = 0; i < ready.players.length; i++) {
+                        let player = ready.players[i];
+                        if (!player.connected) {
```

Comment:
What happens if the player disconnects before a word has been chosen? I think there might be a bug here.

Reply:
What do you think is the bug here?
As without this PR nothing happens when the drawing player disconnects. If he reconnects everything is fine again.
If the time runs out, a new round starts and the dialog will disappear (if the player is now drawing) or refreshes with the name of the new drawer.

Reply:
ah, now i see..
So I will just remove these lines:
`                        if (!player.connected) {
                            continue;
                        }
`